### ISSUE TITLE
Email field signup

### ISSUE THREAD

**Issue by ds6332** [gcivil-nyu-org/team-3-inperson#308]
Email field takes any string in the form "word@word.word".  Maybe a front end check will reduce operations on AWS. Forcing verification before login works

**Comment by sjeremycohen** [comment]
I think you're right that we should be forcing verification before login - @Pankhurinyu @emoltz let's figure out how this can be enforced?

**Comment by emoltz** [comment]
I think we are already forcing verification before login in a way right? Because they can't log in until they click the link in their email.

**Comment by ds6332** [comment]
@emoltz , yeah you are right. I don't know how to enforce email field validation, cause there could be any valid domain and regex structure. But maybe, prevent the email id from being changed. 

What if a user validates with one email id and later changes it to a different email - valid or not.?

**Comment by emoltz** [comment]
Really good point. We would have to build in validation for the changed email too
